### DISCUSSION TITLE
Markers not reachable using keyboard [Accessibility issue]

### DISCUSSION THREAD

**Discussion by nidhimittal90** [d3/d3#3628] [Q&A] [answered]
We are using d3-geo to display worldmap with markers on few countries, while doing accessibility testing using [Accessibility Insights for Web](https://accessibilityinsights.io/docs/web/overview/) extension, none of the markers(countries) are reachable using tab key. This is a serious concern for accessibility as the interactive elements are not reachable through keyboard. Please let me know if there is any fix for this issue.

**Comment by Fil** [comment]
Note that d3-geo is only concerned with creating the proper path for the svg or canvas context.

**Comment by Fil** [comment, marked as answer]
You might want to use tabindex and onfocus to build interactivity.

See for example https://observablehq.com/embed/@fil/svg-tabindex?cells=map
